pkgs/servers/nosql/mongodb/default.nix

```diff
@@ -7,7 +7,7 @@
 
 with stdenv.lib;
 
-let version = "3.2.9";
+let version = "3.4.4";
```

Comment:
actually it should just fail:

    sha256sum mongodb-src-r3.4.4.tar.gz 
    09e962bf3428474b9790bbd464cb6176817f9da6121c30e096240dbb4d51c9f6  mongodb-src-r3.4.4.tar.gz

please update the hash accordingly and test your build!